src/plans/D3MAaveDaiPlan.sol

```diff
@@ -154,19 +145,16 @@ contract D3MAaveDaiPlan is ID3MPlan {
         // Do inverse calculation of interestStrategy
         uint256 variableRateSlope1 = interestStrategy.variableRateSlope1();
         uint256 targetUtil;
-        if (targetInterestRate > _add(base, variableRateSlope1)) {
+        if (targetInterestRate > base + variableRateSlope1) {
             // Excess interest rate
             uint256 r = targetInterestRate - base - variableRateSlope1;
-            targetUtil = _add(
-                            _rdiv(
-                                _rmul(interestStrategy.EXCESS_UTILIZATION_RATE(), r),
-                                interestStrategy.variableRateSlope2()
-                            ),
-                            interestStrategy.OPTIMAL_UTILIZATION_RATE()
-                        );
+            targetUtil = _rdiv(
+                            _rmul(interestStrategy.EXCESS_UTILIZATION_RATE(), r),
+                            interestStrategy.variableRateSlope2()
+                        ) + interestStrategy.OPTIMAL_UTILIZATION_RATE();
```

Comment:
Indentation changed here to 3 spaces instead of 4.
Also the closing bracket of the `_rdiv` is not completely aligned to it (missing one space before the closing bracket).